package/MDAnalysis/coordinates/memory.py

```diff
@@ -514,7 +518,14 @@ def timeseries(self, asel=None, start=0, stop=-1, step=1, order='afc'):
 
         .. versionchanged:: 1.0.0
            Deprecated `format` keyword has been removed. Use `order` instead.
+        .. versionchanged:: 2.4.0
+           Deprecation warning 
         """
+        if stop != -1:
+            warnings.warn("MemoryReader.timeseries inclusive `start` and `stop`"
```

Comment:
Only stop. Start is always inclusive. 

If we mention start here then users might be confused if the meaning of start also changes.